CHANGELOG-3.7.md

```diff
@@ -72,15 +74,26 @@ particular aspect of Ice.
   - [Ruby Changes](#ruby-changes-2)
 - [Changes in Ice 3.7.0](#changes-in-ice-370)
   - [General Changes](#general-changes-6)
-  - [C++ Changes](#c-changes-13)
-  - [C# Changes](#c-changes-14)
+  - [C++ Changes](#c-changes-14)
+  - [C# Changes](#c-changes-15)
   - [Java Changes](#java-changes-5)
   - [JavaScript Changes](#javascript-changes-7)
   - [Objective-C Changes](#objective-c-changes-2)
   - [PHP Changes](#php-changes-4)
   - [Python Changes](#python-changes-6)
   - [Ruby Changes](#ruby-changes-3)
 
+
+# Changes in Ice 3.7.9
+
+These are the changes since Ice 3.7.8.
+
+## C++ Changes
+
+- Fixed a bug in the OpenSSL implementation of IceSSL resulting in the SSL protocol error description not being
+  included with the `Ice::ProtocolException` exception that is thrown when an SSL protocol error is detected during
+  reading. OpenSSL versions previous to `1.1.1e` are not affected.
```

Comment:
The sentence is a bit too long 🙂. Perhaps: 

Fixed an IceSSL bug where the OpenSSL error description was not correctly with the `Ice::ProtocolException` when a read failure occurred.